core-libs/setup/ssr/logger/loggers/express-server-logger.spec.ts

```diff
@@ -0,0 +1,111 @@
+import { Request } from 'express';
+import {
+  DefaultExpressServerLogger,
+  ExpressServerLoggerContext,
+} from './express-server-logger';
+
+describe('ExpressServerLogger', () => {
+  const logger = new DefaultExpressServerLogger();
+  jest.useFakeTimers().setSystemTime(new Date('2023-05-26'));
```

Comment:
[NITPICK]

I'm aware that our default logger is stateless and doesn't depend on third party state. That said, it's just a good practice to re-do a setup of the test for each run separately, to prevent a _potential_ cross-test interference:

(similar comment applies also to other spec files in this PR, where we do `new Xxx()`)

```suggestion
  let logger: DefaultExpressServerLogger;
  
  beforeEach(() => {
    logger = new DefaultExpressServerLogger();
    jest.useFakeTimers().setSystemTime(new Date('2023-05-26'));
  });
```